content/blog/detect-pii/index.md

```diff
@@ -0,0 +1,219 @@
+---
+path: '/detect-pii'
+title: "I shouldn't be seeing this: anonymize sensitive data while debugging using NLP"
+date: 2022-10-10T06:00:00.000+00:00
+featured_image: hero.png
+categories: ['Pixie Team Blogs', 'PII', 'Security']
+authors: ['Benjamin Kilimnik']
+emails: ['benjaminkilimnik@gmail.com']
+---
+
+### Introducing a [PII detector for protocol trace data.](https://detect.streamlitapp.com/) Give it a try and [tell us what you think.](https://github.com/pixie-io/pixie/issues/623) It uses a [custom NLP model](https://huggingface.co/beki/en_spacy_pii_distilbert) trained on a [new PII dataset](https://huggingface.co/datasets/beki/privy) for protocol traces.
+
+![](sample_pii_json.png)
```

Comment:
This image is very similar to the next one (`pii_anonymizer_demo.png`). Do both bring value or should only one be included?

Reply:
Removed this image since it's similar to the first